website/content/api-docs/secret/pki.mdx

```diff
@@ -1602,3 +1602,30 @@ $ curl \
     --data @payload.json \
     http://127.0.0.1:8200/v1/pki/tidy
 ```
+
+# Cluster Scalability
+
+Most non-introspection operations in the PKI secrets engine require a write to 
+storage, and so are forwarded to the cluster's primary for execution.  This table
+outlines which operations can be executed on secondaries and thus scale 
```

Comment:
s/primary/active node/
s/secondaries/performance standbys/

Reply:
Still having trouble with that...